src/main/scala/search/gmosnorth/GmosNorthFilterSelector.scala

```diff
@@ -56,14 +61,36 @@ object GmosNorthFilterSelector {
   private case class Selector(
     disperser: GmosNorthDisperser,
     fpus:      Set[GmosNorthFpu],
-    minNm:     Int,
-    maxNm:     Int,
+    minNm:     Wavelength,
+    maxNm:     Wavelength,
     filter:    Option[GmosNorthFilter],
-    optimalNm: Option[Int] = None
+    optimalNm: Option[Int]
```

Comment:
What is `optimalNm` used for?  Is there a reason not to make it `Option[Wavelength]`?

Reply:
Oh right, duh.

It's not used for anything right now but it seems like we might want to report it to the user or use it in the scoring algorithm maybe? It was in the data provided by science so I didn't want to discard it.